fagiServer/src/main/java/com/fagi/handler/InputHandler.java

```diff
@@ -51,7 +51,9 @@
 
 public record InputHandler(InputAgent inputAgent, OutputAgent out, ConversationHandler conversationHandler, Data data) {
     public void handleInput(Object input) {
-        if (input instanceof TextMessage arg) {
+        if (Objects.isNull(input)) {
```

Comment:
Why use Objects.isNull here? Why not just `input == null`

Reply:
Personal preference. I find it more readable. Especially if the expression is more complicated than this.

Reply:
Changed it to ==